include/fs/fs_interface.h

```diff
@@ -43,6 +43,8 @@ extern "C" {
 
 #ifdef CONFIG_FILE_SYSTEM_FAT
```

Comment:
Are there any blocker for support FAT and NFFS simultaneously?

Reply:
both fat_fs.c and nffs_fs.c will export the same symbols for fs api so it won't link

Reply:
Are there any other obstacles? 

I think about cases like - NFFS for internal storage (e.g. BLE bond data) and FAT for external SD card. For sure  I understand that this is not in scope of this PR intention.

Reply:
we do need to address this, there is a JIRA about this